samples/connectedhomeip/light_bulb/src/LightingManager.cpp

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include "LightingManager.h"
+
+#include <logging/log.h>
+
+#include <drivers/pwm.h>
+#include <zephyr.h>
+
+LOG_MODULE_DECLARE(app);
+
+LightingManager LightingManager::sLight;
+
+int LightingManager::Init(const char * pwmDeviceName, uint32_t pwmChannel)
+{
+    // We use a gpioPin instead of a LEDWidget here because we want to use PWM
```

Comment:
It's actually an outdated comment. Probably forgot to remove it when adding PWM.